packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -254,6 +248,11 @@ const DropdownMenuRootContent = React.forwardRef((props, forwardedRef) => {
           // focus on pointer down, creating the same issue.
           const targetIsTrigger = context.triggerRef.current?.contains(event.target as HTMLElement);
           if (targetIsTrigger) event.preventDefault();
+          else if (!event.defaultPrevented) {
+            const originalEvent = event.detail.originalEvent as PointerEvent;
+            const isModalLeftClick = context.modal && originalEvent.button === 0;
```

Comment:
Do we foresee any issue with this? was easier to move it from `onPointerDownOutside` as we have to account for both modes in this comp.